package.json

```diff
@@ -38,14 +39,16 @@
     "@testing-library/jest-dom": "^6.1.3",
     "@testing-library/react": "^14.0.0",
     "@types/jest": "^29.5.5",
+    "@types/react": "^18.2.22",
+    "@types/react-dom": "^18.2.7",
     "babel-loader": "^8.3.0",
     "babel-plugin-styled-components": "^2.1.4",
     "babel-plugin-transform-react-jsx": "^6.24.1",
     "jest": "^29.7.0",
     "react-test-renderer": "^18.2.0",
     "ts-jest": "^29.1.1",
     "ts-node": "^10.9.1",
-    "typescript": "^5.0.4",
+    "typescript": "^5.2.2",
```

Comment:
Thank you for updating and installing the latest typeScript and including the dev dependencies that we need for our application to run all tests smoothly 🧪🙌🤩🥳✨⭐️